source/articles/_articles.html.slim

```diff
@@ -4,9 +4,7 @@
     - article_category = defined?(category) ? category : nil
 
     - if index == 2
-      .related.related--platform
-        .container
-          h2= link_to "Platform advert goes here", config[:platform_url], target: :blank
+      .related.related--platform data-view-platform-placeholder="true"
```

Comment:
related--platform kinda implies that it's mostly our platform related links? I was thinking that we can use this space creatively - maybe provide ability for partner promotion.

Maybe more generic name? promo-placeholder?